From 6d2724e125aa93afec3907b1579eab254b0dc585 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Mon, 6 Mar 2023 00:07:32 +0100
Subject: [PATCH] ARM: imx: Include on-SoM microSD in list of i.MX6 DHCOM boot
 devices

Add mmc1, which is mapped to optional on-SoM microSD socket,
to the list of distro boot command boot devices.

Upstream-Status: Backport [15db316726eadd3614e8201f67a25723a7d49892]
Signed-off-by: Marek Vasut <marex@denx.de>
---
Cc: Andreas Geisreiter <ageisreiter@dh-electronics.de>
Cc: Christoph Niedermaier <cniedermaier@dh-electronics.com>
Cc: Fabio Estevam <festevam@denx.de>
Cc: Peng Fan <peng.fan@nxp.com>
Cc: Stefano Babic <sbabic@denx.de>
Cc: u-boot@dh-electronics.com
---
 include/configs/dh_imx6.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/dh_imx6.h b/include/configs/dh_imx6.h
index 54b2192b4a8..9d4addbc2f7 100644
--- a/include/configs/dh_imx6.h
+++ b/include/configs/dh_imx6.h
@@ -65,6 +65,7 @@
 
 #define BOOT_TARGET_DEVICES(func) \
 	func(MMC, mmc, 0) \
+	func(MMC, mmc, 1) \
 	func(MMC, mmc, 2) \
 	func(USB, usb, 1) \
 	func(SATA, sata, 0) \
-- 
2.39.2

